cmd/minishift/cmd/start.go

```diff
@@ -126,6 +127,7 @@ func runStart(cmd *cobra.Command, args []string) {
 	setDockerProxy()
 	setOcProxy()
 	setShellProxy()
```

Comment:
At this point I am not sure the purpose if this. So going to keep it . I will dig more and might log an new issue for tracking it.